packages/paste-icons/__test__/__snapshots__/icons.spec.tsx.snap

```diff
@@ -97,6 +97,7 @@ Array [
   "MoreIcon",
   "NeutralIcon",
   "NewIcon",
+  "NoteIcon",
```

Comment:
@pratapdd would you be able to send the Figma file over where this icon is used? There may be another icon we can use here instead that exists in Paste currently.

Reply:
Hi @kelseycavitt 
This is the Figma file https://www.figma.com/file/OwZj3tjpFxFPbe7q5K0OtE/Aion-Components?node-id=740%3A60169

and our UX is `Isabela Razetti`